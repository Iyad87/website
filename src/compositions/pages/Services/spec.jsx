import React from 'react';
import { shallow } from 'enzyme';
import ServicesPage from './index';

test('compositions/pages/Services snapshot', () => {
  const wrapper = shallow(<ServicesPage />);

  expect(wrapper).toMatchSnapshot();
});
