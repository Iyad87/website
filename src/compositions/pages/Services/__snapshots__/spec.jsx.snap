// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/pages/Services snapshot 1`] = `
<React.Fragment>
  <PageSection
    gradient={false}
    gradientHeight="50%"
  >
    <PageHero
      before="services"
      title="Great companies go beyond their customers expectations, over and over again."
    >
      We enable your organisation to deliver world-class technology and user experiences, long after we leave.
    </PageHero>
  </PageSection>
  <PageSection
    gradient={false}
    gradientHeight="50%"
  >
    <ProductList />
    <PageRule />
  </PageSection>
  <PageSection
    gradient={false}
    gradientHeight="50%"
  >
    <ServiceList />
  </PageSection>
  <ServicesTiles />
  <WorkWithUs />
</React.Fragment>
`;
